### PR TITLE
Travis Continuous Integration (CI)

### PR DESCRIPTION
**Pull request by EtienneCmb**
Ci using travis (currently only installation under linux is tested for both python 2 / 3).

Things that need to be done : 
* Git clone mne-data
* Write test so that it can work locally and on travis

### CODE REVIEW

**Comment by EtienneCmb** [comment]
[Here](https://travis-ci.org/neuropycon/ephypype/builds/424030719?utm_source=github_status&utm_medium=notification) you can see that two builds are running. First, a conda env is created, then ephypype is installed and tests (with code coverage) are launched

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+      os: linux
+      python: '3.6'
+
+
+before_install:
+    # Instal miniconda
+    - if [ "${TRAVIS_OS_NAME}" == "linux" ]; then
+        wget http://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh;
+      else
+        wget http://repo.continuum.io/miniconda/Miniconda3-latest-MacOSX-x86_64.sh -O miniconda.sh;
```
Comment (jasmainak) [inline .travis.yml]: you don't need this right? I don't see `mac` in the matrix above ...

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    - hash -r
+    - conda config --set always_yes yes --set changeps1 no
+    - conda update -q conda
+    - conda info -a
+    - SRC_DIR=$(pwd)
+
+
+install:
+    # Create the python environment ;
+    - conda create -q -n testenv python=$TRAVIS_PYTHON_VERSION numpy pip cython matplotlib;
```
Comment (jasmainak) [inline .travis.yml]: why cython?

File: .travis.yml (hunk 1 above)
Comment (EtienneCmb) [reply .travis.yml]: yes, I let it just in case. But if we don't want OSx test, this can be removed

File: .travis.yml (hunk 2 above)
Comment (EtienneCmb) [reply .travis.yml]: yes, my first installation was without cython, but it seems to be needed for mkl. See [this](https://travis-ci.org/EtienneCmb/ephypype/jobs/424025003#L811)

Comment (jasmainak) [reply .travis.yml]: I don't see a directory for docs. I don't know how the docs are being built. It seems the docs are common for all 3 repos. So I don't know how you would do CircleCI here ...

Comment (EtienneCmb) [reply .travis.yml]: Right. Doc is in a separate [directory](https://github.com/neuropycon/neuropycon_doc)

Comment (jasmainak) [reply .travis.yml]: yep. Let's tackle easy stuff first :-)

Comment (EtienneCmb) [reply .travis.yml]: @jasmainak do you have experience with mne data? If so, may be you can increment this PR to download mne-data and cache it?

Comment (jasmainak) [reply .travis.yml]: yep, let me try ...

**Comment by jasmainak** [comment]
@EtienneCmb you made a PR from master :-P

**Comment by EtienneCmb** [comment]
@jasmainak yes, I know. I originally thought that testing a branch might not work without further configuration...

**Comment by jasmainak** [comment]
Merging @EtienneCmb . We can iterate with the next pull requests. Thanks for the contribution !

**Pull request merged**